contracts/src/modules/DummyModule.sol

```diff
@@ -24,11 +27,43 @@ contract DummyModule is BaseModule {
   // address (SafeProtocolManager)
   address manager;
 
+  // address (relayer: keeper, gelato, AA)
+  address public relayer;
+
   // address (Safe address) => DummyConfig
   mapping(address => DummyConfig) public safeConfigs;
 
-  constructor(address _manager, PluginMetadata memory _data) BaseModule(_data) {
+  ////////////////////////////////////////////////////////////////////////////
+  // ERRORS
+  ////////////////////////////////////////////////////////////////////////////
+
+  error UntrustedRelayer(address origin);
+
+  error TooSoon(uint256 currentTime, uint256 updateTime, uint256 minDuration);
+
+  ////////////////////////////////////////////////////////////////////////////
+  // MODIFIERS
+  ////////////////////////////////////////////////////////////////////////////
+
+  modifier onlyRelayer() {
+    if (msg.sender != relayer) revert UntrustedRelayer(msg.sender);
+    _;
+  }
+
+  ////////////////////////////////////////////////////////////////////////////
+  // EVENTS
+  ////////////////////////////////////////////////////////////////////////////
+
+  /// @dev use for subgraph to display basic info in ui as per `safe` basis
+  event PluginTransactionExec(address safe, address gauge, uint256 timestamp);
+
+  constructor(
+    address _manager,
```

Comment:
wondering if it makes sense to save the manager address to `safeConfigs` too, to keep it possible to upgrade the manager for example

(longterm change, outside the scope of this pr obviously)

Reply:
yes, i like the idea of having that as something that is configurable also and gives flexibility in the plugins